tests/nemo_text_processing/zh/test_decimal.py

```diff
@@ -14,17 +14,27 @@
 
 import pytest
 from nemo_text_processing.inverse_text_normalization.inverse_normalize import InverseNormalizer
+from nemo_text_processing.text_normalization.normalize import Normalizer
 from parameterized import parameterized
 
 from ..utils import CACHE_DIR, parse_test_case_file
 
 
-class TestDecimal:
-    inverse_normalizer = InverseNormalizer(lang='zh', cache_dir=CACHE_DIR, overwrite_cache=False)
+class TestDate:
+    normalizer_zh = Normalizer(lang='zh', cache_dir=CACHE_DIR, overwrite_cache=False, input_case='cased')
 
-    @parameterized.expand(parse_test_case_file('zh/data_inverse_text_normalization/test_cases_decimal.txt'))
+    @parameterized.expand(parse_test_case_file('zh/data_text_normalization/test_cases_decimal.txt'))
     @pytest.mark.run_only_on('CPU')
     @pytest.mark.unit
-    def test_denorm(self, test_input, expected):
-        pred = self.inverse_normalizer.inverse_normalize(test_input, verbose=False)
-        assert pred == expected
+    def test_norm_date(self, test_input, expected):
+        preds = self.normalizer_zh.normalize(test_input)
+        assert expected == preds
+
+    # inverse_normalizer = InverseNormalizer(lang='zh', cache_dir=CACHE_DIR, overwrite_cache=False)
+
```

Comment:
same. clean the code

Reply:
This part is responsible for running ITN test. I had separate PRs for TN and ITN, so I commented the ITN part out temporally. This part is needed.

Reply:
Added